laufband/worker_logic.py

```
@@ -0,0 +1,175 @@
+    Returns:
+        Updated worker entry
+
+    Raises:
+        ValueError: If worker not found
+    """
+    if time_provider is None:
+        time_provider = RealTimeProvider()
+
+    worker = session.get(WorkerEntry, worker_id)
+    if worker is None:
+        raise ValueError(f"Worker with identifier {worker_id} not found.")
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)